framework/src/main/groovy/org/moqui/impl/service/runner/EntityAutoServiceRunner.groovy

```diff
@@ -191,6 +191,8 @@ public class EntityAutoServiceRunner implements ServiceRunner {
                 EntityValue svi = efi.makeFind("moqui.entity.SequenceValueItem").condition("seqName", ed.getFullEntityName())
                         .useCache(false).one()
                 logger.warn("======= Got PK violation, current bank is ${bank}, PK is ${newEntityValue.getPrimaryKeys()}, current SequenceValueItem: ${svi}")
+            } else {
```

Comment:
Maybe we should rethrow the exception to make the transaction rollback after write some log.